package.json

```diff
@@ -191,8 +191,10 @@
     "JSONStream": "1.3.5",
     "ansi-regex": "^5.0.1",
     "antlr4ts": "^0.5.0-alpha.3",
-    "archiver": "^5.2.0",
-    "axios": "^1.4.0",
+    "archiver": "^5.3.1",
+    "async": "^3.2.3",
+    "aws4": "^1.12.0",
```

Comment:
__question:__ is it just strangely resolved conflict or we need these new dependencies (`async` and `aws4`) for some reason? Also

Reply:
Good catch. These were newer dependencies from main that should not be here.